src/black/__init__.py

```diff
@@ -224,6 +224,16 @@ def validate_regex(
         "(useful when piping source on standard input)."
     ),
 )
+@click.option(
+    "--python-cell-magics",
+    multiple=True,
```

Comment:
Agreed, though this is how `click` supports multiple options out of the box.  This is also how `target-version` takes multiple arguments.  I expect that we would need to introduce a custom callback to support multiple options, which might then also require custom logic when reading from the `pyproject.toml` file.

My expectation is that this will rather mostly be used in a configuration file and so perhaps the clunkiness on the command line is acceptable.

Reply:
That's fair!